src/install/config/vm.yaml

```diff
@@ -0,0 +1,32 @@
+agentName: vm
+agentType: vuln
+title: 'Import Dependabot data'
+metaDescription: 'Test for vuln mgmt.'
+introFilePath: 'src/install/vm/intro.mdx'
+agentConfigFilePath: 'src/install/config/agent-config/java-newrelic.yml'
+appInfo:
+  - optionType: deployment
+    label: 'Do you want to send new or previously detected vulnerabilities?'
```

Comment:
not sure how it works, but the option for choosing 'new or previously detected' shows up twice in the doc. somehow being doubled